js/modules/k6/crypto/crypto_test.go

```diff
@@ -252,3 +252,48 @@ func TestOutputEncoding(t *testing.T) {
 		assert.EqualError(t, err, "GoError: Invalid output encoding: someInvalidEncoding")
 	})
 }
+
+func TestHMac(t *testing.T) {
+	if testing.Short() {
+		return
+	}
+
+	rt := goja.New()
+	rt.SetFieldNameMapper(common.FieldNameMapper{})
+
+	root, _ := lib.NewGroup("", nil)
+	state := &common.State{Group: root}
+
+	ctx := context.Background()
+	ctx = common.WithState(ctx, state)
+	ctx = common.WithRuntime(ctx, rt)
+
+	rt.Set("crypto", common.Bind(rt, &Crypto{}, &ctx))
+
+	t.Run("Valid", func(t *testing.T) {
```

Comment:
Please add test cases for an invalid algorithm too, and use the `testdata`/`t.Run()` idiom to run hasher + wrapper tests for each supported algorithm: https://github.com/loadimpact/k6/blob/0092f27224b995c3bfd41a7b9cc756dfef8f6777/js/modules/k6/k6_test.go#L49